docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+services:
+  cinema:
+    build:
+      context: .
+
+    env_file:
+        - .env
+
+    ports:
+      - "8001:8000"
+
+    volumes:
+      - ./:/cinema
+
+    command: >
+          sh -c "python manage.py wait_for_db &&
+                 python manage.py migrate &&
+                 python manage.py runserver 0.0.0.0:8000"
+
+    depends_on:
+          - db
+
+  db:
+    image: postgres:16.0-alpine3.17
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - db:$PGDATA
```

Comment:
The volume configuration `db:$PGDATA` might not work as expected if `$PGDATA` is not defined. Consider using a specific path like `db:/var/lib/postgresql/data` to ensure the data is stored correctly.